src/pages/conversion/places-preview-map/utils.js

```diff
@@ -101,25 +101,41 @@ export function getTextStyle(category) {
   return style;
 }
 
+function isValidLatitude(lat) {
+  return lat >= -90 && lat <= 90;
+}
+
+function isValidLongitude(lon) {
+  return lon >= -180 && lon <= 180;
+}
+
+export const getCoordinatesArray = arr => {
+  if (typeof arr[0][0] === 'number') return arr;
+  return getCoordinatesArray(arr[0]);
```

Comment:
This takes only the first array. The rest are being ignored. 
We should flatten them all and return. Not just the first piece.

Reply:
Thanks for the comment. Please check it out now. It is flattening and taking every coordinate now